### DISCUSSION TITLE
Any Idea???

### DISCUSSION THREAD

**Discussion by rufilboss** [FreeOps-Tools/LogIQ#10] [Ideas]
If you have any idea on how we can optimize the development with ease or anything else, please let us know here!

**Comment by Emmauel98** [comment]
Please I'm new, how do I get started

**Reply by rufilboss** [reply]
Hi, you're welcome

**Reply by rufilboss** [reply]
If you'll like to contribute to our project look for existing issue or create new one be fore you started working on it!

**Reply by rufilboss** [reply]
Also, if you've any idea on how we can improve our community and the likes drop it here! You can create issue requesting to join our discord channel
